flash/tabular/classification/data/data.py

```diff
@@ -82,6 +82,15 @@ def __init__(
         dfs = [train_df]
         self._test_df = None
 
+        if categorical_input is None and numerical_input is None:
+            raise TypeError('Both categorical_input and numerical_input are None!')
+
+        if categorical_input is None:
+            categorical_input = []
+
+        if numerical_input is None:
```

Comment:
Why not just have `[]` as default inputs?

Reply:
Yes, you are right @kaushikb11 . one more thing, instead of List shouldn't we have Tuple? Since these values are not going to modify.